BingMaps/sdk-native/TOC.yml

```diff
@@ -74,6 +74,126 @@
       items:
         - name: Map control API classes
```

Comment:
Maybe I'm not familiar enough with what the "TOC.yml" does or what role it plays in order to provide any relevant feedback.  Can you elaborate?

But that seems to be different from the actual API reference index page that corresponds to the documentation page below:
https://docs.microsoft.com/en-us/bingmaps/sdk-native/map-control-api/

Reply:
![image](https://user-images.githubusercontent.com/33468793/183715578-0df5ff85-3a52-48f4-bced-615da29967ea.png)

Reply:
Discussed offline and updated the Table of Contents as a result.